docs/developmentSetup.md

```diff
@@ -0,0 +1,44 @@
+# developmentSetup
+
+Created By: Alexis Ortiz Ojeda
+Last Edited: Feb 21, 2020 11:32 PM
+
+# Welcome to Development Setup!
+
+If you are reading this you are ready to contribute to c0d3.com!
+
+To begin we will need to set up the code base on your own machine. To do this we will be creating a copy of the **c0d3** GitHub repository which can be found [here](https://github.com/garageScript/c0d3-app).
```

Comment:
To begin we will need to set up the code base on your own machine. To do this we will be creating a copy of the [**c0d3** GitHub repository](https://github.com/garageScript/c0d3-app).